ja/documentation/installation/index.md

```diff
@@ -34,7 +34,7 @@ Windows 10 では、[Windows Subsystem for Linux][wsl] を使用して、サポ
   * [Arch Linux](#pacman)
   * [macOS](#homebrew)
   * [FreeBSD](#freebsd)
-  * [OpenBSD](#openbsd)
+  * [OpenBSD](#doas)
```

Comment:
Can you add `#doas` to OpenBSD section same as https://github.com/ruby/www.ruby-lang.org/pull/3103/files#diff-22f637eb1eb70979587a5c96614fddb46bdf7ae2d34a04ee30915ba526e76fd1R175?

Reply:
The `#doas` has been in OpenBSD section (ja).

I first saw `#doas` in OpenBSD section (ja), and I saw the most of the OS list  use the package manager name as link. So I use the `#doas` for OpenBSD in the PR https://github.com/ruby/www.ruby-lang.org/pull/3103/files#diff-22f637eb1eb70979587a5c96614fddb46bdf7ae2d34a04ee30915ba526e76fd1R175, and this PR.

I just found it use `#openbsd` in (en) https://github.com/ruby/www.ruby-lang.org/blob/833920128596ef7aa9d766c01de3e857b16d448c/en/documentation/installation/index.md?plain=1#L47 , would it be better to follow the (en) to use `#openbsd`?

Reply:
>The #doas has been in OpenBSD section (ja).

Thanks. I missed it.

Reply:
>would it be better to follow the (en) to use #openbsd?

+1

Reply:
I updated the links to follow (en) version.
I also update the PR: #3103 

Thank you.